CryptoTechReminderSystem/UseCase/GetLateDevelopers.cs

```diff
@@ -61,5 +61,11 @@ private static DateTimeOffset GetEndingDate(DateTimeOffset currentDateTime)
             var daysToFriday = (7 + (DayOfWeek.Friday - currentDateTime.DayOfWeek)) % 7;
             return currentDateTime.AddDays(daysToFriday);
         }
+
```

Comment:
Doest his negate the need for email matching, are we now matching by name? Maybe we should handle the `.co.uk` and `.com` exceptions by finding the index further down so we are certain that they have a madetech email?